app/Filament/Resources/HubResource/RelationManagers/ParentHubsRelationManager.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Filament\Resources\HubResource\RelationManagers;
+
+use App\Filament\Resources\HubResource;
+use App\Models\GameSet;
+use App\Platform\Actions\AttachGameLinksToGameSetAction;
+use App\Platform\Actions\DetachGameLinksFromGameSetAction;
+use App\Platform\Enums\GameSetType;
+use Filament\Forms;
+use Filament\Resources\RelationManagers\RelationManager;
+use Filament\Tables;
+use Filament\Tables\Table;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Collection;
+
+class ParentHubsRelationManager extends RelationManager
+{
+    protected static string $relationship = 'parents';
+    protected static ?string $title = 'Related Hubs';
+    protected static ?string $icon = 'heroicon-m-link';
+
+    public static function getBadge(Model $ownerRecord, string $pageClass): ?string
+    {
+        $count = $ownerRecord->parents->count();
+
+        return $count > 0 ? "{$count}" : null;
+    }
+
+    public function table(Table $table): Table
+    {
+        return $table
+            ->columns([
+                Tables\Columns\ImageColumn::make('badge_url')
+                    ->label('')
+                    ->size(config('media.icon.sm.width'))
+                    ->url(function (GameSet $record) {
+                        if (request()->user()->can('manage', GameSet::class)) {
+                            return HubResource::getUrl('view', ['record' => $record]);
+                        }
+                    }),
+
+                Tables\Columns\TextColumn::make('parent_game_set_id')
+                    ->label('Hub ID')
+                    ->sortable()
+                    ->searchable()
+                    ->url(function (GameSet $record) {
+                        if (request()->user()->can('manage', GameSet::class)) {
+                            return HubResource::getUrl('view', ['record' => $record]);
+                        }
+                    }),
+
+                Tables\Columns\TextColumn::make('title')
+                    ->label('Title')
+                    ->sortable()
+                    ->searchable()
+                    ->url(function (GameSet $record) {
+                        if (request()->user()->can('manage', GameSet::class)) {
+                            return HubResource::getUrl('view', ['record' => $record]);
+                        }
+                    }),
+            ])
+            ->filters([
+
+            ])
+            ->headerActions([
+                Tables\Actions\Action::make('add')
+                    ->label('Add related hubs')
```

Comment:
Adding a related hub generates an audit message with "field 0":
![image](https://github.com/user-attachments/assets/9c31355d-cd6a-4759-a5cb-e2a6babf7464)

"field parents" is displayed when removing it
![image](https://github.com/user-attachments/assets/dcdfdac0-ab94-4c36-ab6f-a6a9740ae3e7)

Reply:
Looks like I had a typo in the model: a `=>` was mistyped as `=`. Fixed in latest.